packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -14,7 +14,7 @@ import { ComboboxAutocomplete } from "./combobox.options.js";
  * @public
  */
 export type ComboboxOptions = StartEndOptions & {
-    indicator?: string | SyntheticViewTemplate;
+    openIcon?: string | SyntheticViewTemplate;
```

Comment:
This doesn't necessarily signify an open state, I think this is likely more confusing than `indicator`. Some implementations may swap or toggle this while others may keep it static. I'd prefer to keep the name more generic rather than related to a given state of this control.

Reply:
"Open" in this case was an action not a state. Like where you would click to open the combobox. "Indicator" I reserved for states, like a "checked" or "indeterminate" indicator. I matched the name to the state in the component, so "open" here because the action it triggers sets the open property. On a menu it's "submenu", on others it's "expand-collapse".

I avoided simply "icon" even for components where there is only one to avoid confusion with common start and end slot usage.

Reply:
I think the confusion is that there is not alternative here for the state, so it implies that there should be. In scenarios where there is a singular application I think the most intuitive anatomy doesn't map to a state.

Reply:
Just commenting here so it travels along that I don't know that any of these *should* be required. My preference here is that its easier to understand the UX if the icon is there, but I don't know that an element is required in all instances of all of these controls. Someone may decide it's easier to style it a different way or address it uniquely. Making any of these required may help some but it also adds a barrier of entry for others who don't want to provide it and end up having to do so.

Reply:
We've resolved the pattern for providing icons.
Regarding the name, I've used the dual-name pattern here , openCloseIcon, following expandCollapseIcon from other components. This feels like something we can apply consistently.
I've reserved "indicator" for elements that represent a persistent state like "checked".